react/frontpage/EventsContent.jsx

```diff
@@ -0,0 +1,129 @@
+// EventsContent.jsx
+var React = require('react');
+
+var Modal = require('react-bootstrap').Modal;
+var Input = require('react-bootstrap').Input;
+var FormControls = require('react-bootstrap').FormControls;
+var Button = require('react-bootstrap').Button;
+if (typeof Button === 'undefined' || Button === null) {
+    console.log("issues!!\n");
+}
+
+var EventsContent = React.createClass({
+	render: function() {
+		return(
+			 <div className='wf-container'>
+			{ JSON.parse(this.props.data).events.map (function(el) {
+					return (
+						<div className="monthEvents" key={el['month']}>
+							<h1>{el['month']}</h1>
+							<div className="eventBlock">
+								{el['arr'].map( function(event){
+									return (
+										<Event event={event} key={event['id']}/>
+									);
+								})
+								}
+							</div>
+						</div>
+					);
+				})
+			}
+			</div>
+		);
+	}
+});
+
```

Comment:
I should start doing this too-- could you write a comment for what each react class does right before defining them?